posts/2020/03/Spyder-4.md

```diff
@@ -0,0 +1,48 @@
+I decided to wait until now to write a blogpost about it because shortly after the initial release, we found several critical performance issues and some regressions with respect to Spyder 3, most of which are fixed now in [version 4.1.2](https://github.com/spyder-ide/spyder/releases/tag/v4.1.2), released on April 3rd 2020.
+
+<!-- TEASER_END -->
+
+This new release comes with a lengthy list of user-requested features aimed at providing an enhanced development experience at the level of top general-purpose editors and IDEs, while strengthening Spyder's specialized focus on scientific programming in Python.
+The interested reader can take a look at some of them in [previous](https://labs.quansight.org/blog/2019/11/variable-explorer-improvements-in-Spyder-4/) [blog](https://labs.quansight.org/blog/2019/11/File-management-improvements-in-Spyder4/) [posts](https://labs.quansight.org/blog/2019/08/spyder-40-beta4-kite-integration-is-here/), and in detail in our [Changelog](https://github.com/spyder-ide/spyder/blob/master/CHANGELOG.md#version-400-2019-12-06).
+However, this post is not meant to describe those improvements, but to acknowledge all people that contributed to making Spyder 4 possible.
+
+Spyder 4 is the first version developed with financial support from multiple companies, as well as donations by the international user community.
+However, as a project, we couldn't have been able to reach the level of maturity needed to receive and handle that support without the pivotal opportunities Travis Oliphant, former CEO of Anaconda and current leader of Quansight, gave me to work in Scientific Python.
+Thanks to him, I became part of Anaconda Inc. in 2015; I was able to hire a small small team to improve Spyder within Anaconda in late 2016; and I was hired by Quansight to work solely on Spyder in 2018.
+As with other projects in our ecosystem, such as [Bokeh](https://github.com/bokeh/bokeh), [Dask](https://github.com/dask/dask) and [Numba](https://github.com/numba/numba), Spyder benefited immensely from Travis' trust in the role these efforts and ours could play in the future.
+He certainly believed in the vision their maintainers worked so hard to make a reality, even if their beginnings were humble and their chances of survival uncertain.
+Therefore, my first big acknowledgment is to Travis: thanks for giving us a chance!
+
+I also want to thank our community for its continued support.
+As I've witnessed during my years as Spyder's lead developer, many newbies and veterans alike keep choosing Spyder as their primary tool for scientific programming in Python.
+It's really you, the members of this fantastic community, which keep Spyder relevant in a highly competitive field by using and contributing back to it.
+I'm also honored and humbled to have received such first-class additions to Spyder 4 from our volunteer contributors as significant improvements to our documentation, thanks to [CAM Gerlach](https://github.com/CAM-Gerlach).
```

Comment:
It seems kinda strange to have just my name singled out up here, and everybody else down there in one big blob; Jitse, Brian, Quentin, and Jean-Sebastian were volunteer contributors and dev team members just like I was, and the syntax sounds weird only listing one person.

Reply:
I agree. I'll do this change.